.pre-commit-config.yaml

```diff
@@ -77,27 +77,6 @@ repos:
             --linelength=88,
             '--filter=-readability/casting,-runtime/int,-build/include_subdir,-readability/fn_size'
         ]
--   repo: https://github.com/pycqa/pylint
```

Comment:
Don't think this should be removed?

Reply:
yes... I remove pylint temporary when running `pre-commit run --color=always --hook-stage manual --all-files`, but I should find a better way.

I've updated a new version again.